HACKING

```diff
@@ -699,9 +699,13 @@ It can be either long or short, depending on what it tests.
 
 The expected output should be in the file ``tests/ctags/mytest.ext.tags``
 (which is the same name as the source, but with ``.tags`` appended), and
-should be in the format generated by ``geany -g``. This file contains
-the tag information expected to be generated from the corresponding
-source file.
+should be in the format generated by::
+
+    $ geany -g tmp.ext.tags tests/ctags/mytest.ext
+    $ cat tmp.ext.tags | scripts/print-tags.py > tests/ctags/mytest.ext.tags
```

Comment:
nitpick: could use `scripts/print-tags.py < tmp.ext.tags > tests/ctags/mytest.ext.tags` and drop the :cat:

Reply:
Right. Despite the fact that 🐈s are great, this one can be dropped. I'll merge the 🐱less version.